Symfony/CS/Tests/Fixer/PSR2/EofEndingFixerTest.php

```diff
@@ -18,6 +20,13 @@
  */
 class EofEndingFixerTest extends AbstractFixerTestBase
 {
+    public function testOrder()
```

Comment:
We already have a test method for it, please use it instead to keep it in one place:
https://github.com/SpacePossum/PHP-CS-Fixer/blob/master/Symfony/CS/Tests/FixerTest.php#L194